README.md

```diff
@@ -1,96 +1,354 @@
-# @govtechsg/oa-verify
-
 [![CircleCI](https://circleci.com/gh/Open-Attestation/oa-verify.svg?style=svg)](https://circleci.com/gh/Open-Attestation/oa-verify)
 
-Library to verify any [OpenAttestation](https://github.com/Open-Attestation/open-attestation) document. This library implements [the verifier ADR](https://github.com/Open-Attestation/adr/blob/master/verifier.md).
+# Open Attestation (Verify)
+
+The [Open Attestation (Verify)](https://github.com/Open-Attestation/oa-verify) repository is the codebase for the npm module that allows you to verify [wrapped document](/docs/developer-section/libraries/open-attestation#wrapping-documents) programmatically. This is useful if you are building your own API or web components. Some common use cases where you will need this module:
+
+- [Verifying a document](#verifying-a-document)
+- [Building custom verifier](#custom-verification)
+- [Building custom validation](#custom-validation)
+
+This module does not provide the following functionality:
+
+- Programmatic wrapping of OA documents (refer to [Open Attestation](https://www.openattestation.com/docs/developer-section/libraries/open-attestation#wrapping-documents))
+- Encryption or decryption of OA documents (refer to [Open Attestation (Encryption)](https://www.openattestation.com/docs/developer-section/libraries/open-attestation-encryption))
+- Programmatic issuance/revocation of document on the Ethereum blockchain
 
 ## Installation
 
-```sh
-npm install @govtechsg/oa-verify
+```bash
+npm i @govtechsg/oa-verify
 ```
 
+---
+
 ## Usage
 
-```typescript
-import { documentRopstenValidWithToken } from "./test/fixtures/v2/documentRopstenValidWithToken";
-import { verify, isValid } from "@govtechsg/oa-verify";
+### Verifying a document
 
-const fragments = await verify(documentRopstenValidWithToken);
-console.log(fragments); // see below
-console.log(isValid(fragments)); // display true
-```
+A verification happens on a wrapped document, and it consists of answering to some questions:
+
+- Has the document been tampered with ?
+- Is the issuance state of the document valid ?
+- Is the document issuer identity valid ? (see [identity proof](https://www.openattestation.com/docs/docs-section/how-does-it-work/issuance-identity))
+
+A wrapped document (shown below) created using [Open Attestation](https://www.openattestation.com/docs/developer-section/libraries/open-attestation) would be required.
 
 ```json
-[
-  {
-    "data": true,
-    "name": "OpenAttestationHash",
-    "status": "VALID",
-    "type": "DOCUMENT_INTEGRITY"
-  },
-  {
-    "data": {
-      "details": [
-        {
-          "address": "0xe59877ac86c0310e9ddaeb627f42fdee5f793fbe",
-          "minted": true
+{
```

Comment:
Just to make sure, is this really a valid document that works on ropsten? If that's the case can add that sentence just above? (below you will find a valid document issued on ropsten, etc.)